app/javascript/react/pages/Home/Home.js

```diff
@@ -19,7 +35,14 @@ const Home = () => {
   const addInputElementHandler = (e) => {
     e.preventDefault();
     if (nameInput || valueInput) {
-      // send item through graphql query
+      const data = {
```

Comment:
should we construct the data field like this? could we not simply render the existing data as key value pairs on the form and have the form update save all the key value's? so just have

```
const [dailyStatData, setDailyStatData] = useState({});

...

# set a date
# fetch daily stats for date
# render fields 
```

hmm maybe that is next up

then we wouldn't need the whole valueInput and nameInput but I'll give a pass on this maybe

Reply:
yes that was my next step, initial step was to create a POC to add or update value using GQL and form.